### DISCUSSION TITLE
Why is there no package.json file in the repository where the clone button is clicked？

### DISCUSSION THREAD

**Discussion by chaos-zhu** [vercel/community#1929] [Help] [answered]
### Summary

I tried to click the Deploy button and then create a private library on the vercel page, but there was no package.json file when deploying

https://github.com/chaos-zhu/ruanyf-weekly
![image](https://user-images.githubusercontent.com/44160914/229014132-8fb6aaa0-af9a-45d8-87df-5c08d46fc6ea.png)

Then click the create button
![image](https://user-images.githubusercontent.com/44160914/229014444-aa5e10dd-e173-438f-b764-97c1e4525778.png)

![image](https://user-images.githubusercontent.com/44160914/229013721-0cc5f9f8-645f-4209-b650-7adb12320ac1.png)

And clone's repository doesn't have package.json files

![image](https://user-images.githubusercontent.com/44160914/229014337-34ef8d19-a260-48f4-8ec1-d7ade87b7456.png)

**Comment by amyegan** [comment, marked as answer]
Hi @chaos-zhu! It looks like all `.json` files are being skipped because of this line in the `.gitignore`:
https://github.com/chaos-zhu/ruanyf-weekly/blob/a6a3629745d08282efc6a25e36a544b1eb3b8fd1/.gitignore#L114

**Reply by chaos-zhu** [reply]
It's working very well. thank you😊
